.azure/gpu-tests-pytorch.yml

```diff
@@ -108,19 +108,20 @@ jobs:
 
     - bash: pip uninstall -y -r requirements/pytorch/strategies.txt
       condition: eq(variables['scope'], '')
-      displayName: 'UnInstall strategies'
+      displayName: 'Uninstall strategies'
 
     - bash: |
         set -e
         CUDA_VERSION_BAGUA=$(python -c "print([ver for ver in [116,113,111,102] if $CUDA_VERSION_MM >= ver][0])")
         pip install "bagua-cuda$CUDA_VERSION_BAGUA"
 
-        CUDA_VERSION_MM_COLOSSALAI=$(python -c "import torch ; print(''.join(map(str, torch.version.cuda)))")
-        CUDA_VERSION_COLOSSALAI=$(python -c "print([ver for ver in [11.3, 11.1] if $CUDA_VERSION_MM_COLOSSALAI >= ver][0])")
-        pip install "colossalai==0.1.12+torch${PYTORCH_VERSION}cu${CUDA_VERSION_COLOSSALAI}" --find-links https://release.colossalai.org
-
         pip install -r requirements/pytorch/strategies.txt --find-links ${TORCH_URL}
 
+        # TODO: Lift the restriction after
+        # https://github.com/microsoft/DeepSpeed/commit/b587c7e85470329ac25df7c7c2521ff9b2833db7
+        # gets released
+        pip install "deepspeed<0.8.0"
```

Comment:
lets pin it rather here: https://github.com/Lightning-AI/lightning/pull/16747/files#r1105178273